contracts/src/registry/ETHRegistrar.sol

```diff
@@ -0,0 +1,52 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.13;
+
+import {IETHRegistrar} from "./IETHRegistrar.sol";
+import {IRegistry} from "./IRegistry.sol";
+import {IERC1155Singleton} from "./IERC1155Singleton.sol";
+import {IETHRegistry} from "./IETHRegistry.sol";
+
+contract ETHRegistrar is IETHRegistrar {
+    IETHRegistry public registry;
+    mapping(address => bool) public controllers;
+    mapping(uint256 => bool) public registered;
```

Comment:
Is this necessary?

Reply:
Apologies, was planning to remove this but then I got caught up in a merge.